homeworks/07-mars-viewer/src/reducer/changeSolutionReducer.ts

```diff
@@ -0,0 +1,20 @@
+import {MarsViewerState} from '../types/state'
+import {MarsViewerAction, MarsViewerActionType} from '../types/action'
+
+
+export const changeSolutionReducer = (state: MarsViewerState, action: MarsViewerAction) => {
+    switch (action.type) {
+        case MarsViewerActionType.LOADED_NEW:
+            return {...state, imagesLinks: action.payload, loaded: true, loading: false}
+        case MarsViewerActionType.LOADING_NOW:
+            return {...state, imagesLinks: [], loaded: false, loading: true, currentSelectedSolution: action.payload}
```

Comment:
according to task description loaded data should be kept in store after loading, here it clear after input value change